tests/e2e/utils/src/components.js

```diff
@@ -435,6 +435,40 @@ const createCoupon = async ( couponAmount = '5', discountType = 'Fixed cart disc
 	return couponCode;
 };
 
+/**
+ * Update an order.
+ *
+ * @param noticeText The text that appears in the notice after updating the order.
+ */
+const updateOrder = async ( noticeText ) => {
+	// Wait for auto save
+	await page.waitFor( 2000 );
```

Comment:
This inserts a 2 second delay to every call. Could we add a bool parameter for  wait flag?

Reply:
Sure thing! Admittedly I copied this from another similar method, but I like the idea of making it opt-in.